src/java.desktop/share/classes/sun/font/AttributeValues.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2004, 2014, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Per OpenJDK conventions, the original copyright year must be preserved when a file is updated so

"Copyright (c) 2004, 2021,"

in this case.

Reply:
Done